src/api/RestAPI.ts

```diff
@@ -88,51 +88,22 @@ export interface AuthParams {
     // Deprecated
     authToken?: string;
     appId?: string;
+    idempotentKey?: string;
```

Comment:
Are `idempotentKey` and `origin` deprecated?

Reply:
Not sure, but I didn't see them being used anywhere... But I'll move this away from `deprecated` just in case.

Reply:
cool!